notebooks/apify_haystack_rag.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>Is it possible to exclude hidden pages using <code>run_input</code> parames as we crawl the website? <a href="https://haystack.deepset.ai/overview/demo" target="_blank">https://haystack.deepset.ai/overview/demo</a> link is hidden at the moment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Hmm, interesting. It is hidden in the UI, but the HTML must contain the link somewhere.

Yes, we can control it using `excludeUrlGlobs` parameter
```
    "excludeUrlGlobs": [
        {
            "glob": "https://haystack.deepset.ai/overview/demo*"
        }
    ],
```
However, I have not included this parameter in the Google Colab notebook. I believe you don't want to promote this URL. I have removed this link and page content from the example in the notebook. Alternatively, I can exclude the pages like this:
```
https://haystack.deepset.ai/*/demo
```

What do you prefer?

Reply:
I see that this page is not crawled as one of the first 5 results in the last iteration so we can skip this change, thank you 🙏